src/xref2/ref_tools.ml

```diff
@@ -349,7 +349,7 @@ module L = struct
             when name = LabelName.to_string name' ->
               Ok (`Identifier label)
           | _ -> find tl)
-      | [] -> Error (`Find_by_name (`Page, name))
+      | [] -> Error (`Find_by_name (`Label, name))
```

Comment:
Yes, it could be made in another PR. Not sure that's worth it, given how small it is! But if anyone confirm it's helpful, I'll separate it into another PR!